blueprint/core/blueprint_TimeoutFunctionManager.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+template <typename Error>
+struct TimeoutFunctionManager : private juce::MultiTimer
+{
+    ~TimeoutFunctionManager() override {}
+
+    void reset()
+    {
+        JUCE_ASSERT_MESSAGE_THREAD
+
+        for(const auto &[id, timer] : timeoutFunctions)
+        {
+            stopTimer(id);
+        }
+        timeoutFunctions.clear();
+    }
+
+    juce::var clearTimeout(const juce::var::NativeFunctionArgs& args)
```

Comment:
With the API cleanup mentioned above, this could just be

```
juce::var clearTimeout(int timerId);
```

Delegate all of the error checking + stuff to the helper function for binding